src/components/ProfileDisplay.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Paper from '@material-ui/core/Paper';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import CommunityGrid from './CommunityGrid';
+import StudentGrid from './StudentGrid';
+
+export default function ProfileDisplay() {
+  const [user, setUser] = React.useState(0);
```

Comment:
a `user` value of `0` does not make sense. Either set `user` to something meaningful (like a string) or change the name of the state variable to something where `0` makes sense as a value.

Reply:
This no longer exist as I did the recommended change to use `<TabPanel>`